src/tests/ftest/datamover/dm_posix_meta_entry.py

```diff
@@ -95,13 +95,13 @@ def run_dm_posix_meta_entry(self, tool):
             cmp_times=preserve_on, cmp_xattr=preserve_on)
 
         # DAOS -> POSIX
-        posix_dst_path = self.new_posix_test_path(False, posix_root)
+        posix_dst_path = self.new_posix_test_path(parent=self.workdir)
         self.run_datamover(
             test_desc + "(DAOS->POSIX)",
             "DAOS", daos_src_path, pool1, cont1,
             "POSIX", posix_dst_path)
         self.compare_data(
-            dfuse_src_path, posix_dst_path,
+            dfuse_src_path, posix_dst_path + daos_src_path,
```

Comment:
Just FYI, this only works because daos_src_path is only 1 directory deep. E.g. `/dir0`. Also, if wanted, you could pass `create=False` to `new_posix_test_path` above so the ultimate destination is just `posix_dst_path` instead of `posix_dst_path + daos_src_path`

Reply:
@daltonbohning yeah I checked the contents of posix_dst_path before adding daos_src_path so I knew there was only one directly in it. It does not work to pass create=False to new_posix_test_path though, and causes test errors.

Reply:
Hmm... Maybe I'm forgetting something (it's been a while). But this IS working, so that's okay

Reply:
@daltonbohning yeah I was wrong I added the create=False to the src instead of dst passed when I tested it. I changed to this way since it is simpler.